app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb

```diff
@@ -0,0 +1,13 @@
+module Spree
```

Comment:
@chhlga please move this file into `app/serializerls/spree_multi_vendor/v2/storefront/stock_location_serializer_decorator.rb`